src/usercontent/index.ts

```diff
@@ -11,10 +11,11 @@ function remoteRender(event) {
     const img = document.createElement("span"); // we'll mask it as an image
```

Comment:
types

Reply:
TS seems to be able to infer the type based on the parameter

Reply:
I believe that's your IDE, not typescript.

Reply:
Ah, I guess that could be the case, added the types into the code